packages/host/tests/integration/components/serialization-test.gts

```diff
@@ -4416,4 +4418,146 @@ module('Integration | serialization', function (hooks) {
       });
     });
   });
+
+  module('base cards', function () {
```

Comment:
I added a custom block within serialization test. This is particurly useful for base cards with custom serialize and deserialize